xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -239,9 +240,21 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates whether this <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> is collectible.</summary>
+        <value><see langword="true" /> if this <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> is collectible; otherwise, <see langword="false" />. If a value is not specified in the constructor, the value is <see langword="false" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ A collectible AssemblyLoadContext may be collected by the garbage collector anytime the Object is unreachable.
+
+ Care should be taken to avoid holding references in global data structures which may unintentionally prevent collection.
+
+ Similarly, care should also be taken to hold references when intentionally trying to prevent garbage collection.
```

Comment:
@janvorli I was tempted to add a comment here that threads running in a collectible ALC should hold a reference in the stack, but I was sure what the guidance was supposed to be.

Reply:
I am not sure what you are trying to say. What reference they should hold in the stack?

You can take a look at my pending https://github.com/dotnet/docs/pull/10256/files, section "Troubleshoot unloadability issues" for a list of things that can hold collectible context alive.

Reply:
I just revised this to refer to your new doc.  Thanks.